includes/admin/welcome.php

```diff
@@ -237,20 +367,69 @@ public function about_screen()
+				<div
+					class="simcal-mt-[121px] simcal-w-[100%] simcal-h-[auto] simcal-bg-sc_green-100 simcal-rounded-[20px] simcal-pl-[5%]">
+					<div class="simcal-pt-[62px] simcal-font-bold simcal-text-xl simcal-text-sc_black-200">
+						<?php _e("Simple Calendar is created by a worldwide team of developers. If you'd ", 'google-calendar-events'); ?>
+					</div>
+					<div class="simcal-text-xl simcal-text-sc_black-200">
+						<?php _e(
+      	"like to contribute please visit our <a href='%s' class='simcal-text-sc_green-200' target='_blank'>GitHub repo. </a>",
+      	'google-calendar-events'
+      ); ?>
+					</div>
+					<div class="simcal-flex simcal-pt-[20px]">
+						<div class="simcal-w-[60%]">
+							<?php echo $this->contributors(); ?>
+						</div>
+						<div class="simcal-w-[40%]">
+							<div
+								class="simcal-mt-[20px] simcal-mr-[5%] simcal-w-[83%] simcal-h-[366px] simcal-rounded-[20px] simcal-bg-white ">
+								<div class="simcal-pl-[42%] ">
+									<img class="simcal-pt-[19px]" src="<?php echo $image_about_path . '/rating.png'; ?>" />
+								</div>
+								<div class="simcal-mt-[15px] simcal-text-center simcal-font-semibold simcal-text-lg ">
+									<Span>
+										<?php _e('Please Rate Us !', 'google-calendar-events'); ?>
+									</Span>
+								</div>
+								<div
+									class="simcal-mt-[5px] simcal-text-center simcal-font-normal simcal-text-base simcal-text-gray-500">
+									<Span>
+										<?php _e('If you like Simple Calendar please Rate Us', 'google-calendar-events'); ?>
+									</Span>
+								</div>
+								<div class="simcal-mt-[44px]">
+									<?php // Rating function is used here
+
+		sc_rating(); ?>
+								</div>
+								<a
+									href="https://wordpress.org/support/plugin/google-calendar-events/reviews/?filter=5#new-post">
+									<button type="button"
+										class="simcal-mt-[20px] simcal-m-auto simcal-flex simcal-justify-center simcal-items-center simcal-w-[80%] simcal-h-[40px] simcal-bg-sc_green-200 simcal-text-white simcal-text-xl simcal-font-medium simcal-rounded-md">
+										<?php _e('Rate Now', 'google-calendar-events'); ?>
+									</button>
+								</a>
+								<div class="simcal-mt-[25px] simcal-text-center simcal-text-base simcal-text-sc_blue-200">
+									<a class="hover:simcal-text-sc_green-200"
+										href="https://wordpress.org/support/plugin/google-calendar-events/reviews/?filter=5">
+										<?php _e('See All Customers Reviews', 'google-calendar-events'); ?>
+									</a>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div>
 				<p class="about-description">
-					<?php printf(
-     	__(
-     		"Simple Calendar is created by a worldwide team of developers. If you'd like to contribute please visit our <a href='%s' target='_blank'>GitHub repo</a>.",
-     		'google-calendar-events'
-     	),
-     	simcal_get_url('github')
-     ); ?>
+					<?php printf(__('', 'google-calendar-events'), simcal_get_url('github')); ?>
```

Comment:
There is empty string, what is this for?

Reply:
i had removed it.